src/autora/doc/pipelines/main.py

```diff
@@ -107,22 +109,21 @@ def eval_prompt(data_file: str, pred: Predictor, prompt: str, param_dict: Dict[s
     bleu, meteor = eval_bleu_meteor(predictions, labels)
     semscore = eval_semscore(predictions, labels)
     pred_time = timer_end - timer_start
-    prompt_hash = hash(prompt)
     mlflow.log_metric("prediction_time/doc", pred_time / (len(inputs)))
     for i in range(len(inputs)):
-        mlflow.log_text(labels[i], f"{prompt_hash}_label_{i}.txt")
-        mlflow.log_text(inputs[i], f"{prompt_hash}_input_{i}.py")
-        mlflow.log_text(predictions[i], f"{prompt_hash}_prediction_{i}.txt")
+        mlflow.log_text(labels[i], f"prompt_{prompt_index}_label_{i}.txt")
```

Comment:
It just occurred to me that the `input` and `label` artifacts will be the same for all prompts, so they don't need to be repeated here, right? Can you refactor this so that only the prediction is logger per-prompt?

Reply:
Done.